ui/src/shared/components/Dygraph.js

```diff
@@ -160,7 +154,9 @@ export default class Dygraph extends Component {
       colors: this.getLineColors(),
       series: this.hashColorDygraphSeries(),
       plotter: isBarGraph ? barPlotter : null,
-      visibility: this.visibility(),
+      drawCallback: () => {
+        this.annotationsRef.setState({lastUpdated: Date.now()})
+      },
```

Comment:
instead of passing all of annotationsref and all its powers- couldn't we pass a function that can only set lastupdated  state? seems neater that way.

Reply:
I will expose a `heartbeat` function on `this.annotationsRef`.